docs/howto/openstack/nova/new-vgpu-server.md

```diff
@@ -0,0 +1,210 @@
+        --boot-from-volume 32 \
+        --network cc-net \
+        --security-group default \
+        --key-name mykey \
+        --wait \
+        my-vgpu-server
+    ```
+        
+    Before you check whether the server does have virtual GPU support, we
+    recommend assigning a floating IP to it.
+
+## Checking the status of vGPU support
+
+From your local computer, connect to the new remote server via SSH.
+Type, for instance:
+
+```bash
+ssh ubuntu@<floating_ip_of_new_server>
+```
+
+Alternatively, don't bother with floating IPs and connect using
+`openstack`:
+
+```bash
+openstack server ssh my-vgpu-server -- -l ubuntu
+```
+
+Upon successful connection to the server, in your terminal window you
+will see something like this:
+
+```plain
+Welcome to Ubuntu 22.04.2 LTS (GNU/Linux 5.15.0-78-generic x86_64)
+
+ * Documentation:  https://help.ubuntu.com
+ * Management:     https://landscape.canonical.com
+ * Support:        https://ubuntu.com/advantage
+
+  System information as of Sun Aug 13 06:11:14 UTC 2023
+
+  System load:  0.0                Processes:             159
+  Usage of /:   17.2% of 29.90GB   Users logged in:       0
+  Memory usage: 1%                 IPv4 address for ens3: 10.254.254.216
+  Swap usage:   0%
+
+ * Strictly confined Kubernetes makes edge and IoT secure. Learn how MicroK8s
+   just raised the bar for easy, resilient and secure K8s cluster deployment.
+
+   https://ubuntu.com/engage/secure-kubernetes-at-the-edge
+
+Expanded Security Maintenance for Applications is not enabled.
+
+0 updates can be applied immediately.
+
+Enable ESM Apps to receive additional future security updates.
+See https://ubuntu.com/esm or run: sudo pro status
+
+The list of available updates is more than a week old.
+To check for new updates run: sudo apt update
+
+Last login: Sun Aug 13 05:48:57 2023 from 198.51.100.129
+```
+
+Your first clue regarding proper vGPU support should be in the output of
+the `lspci` command:
+
+```bash
+lspci
+```
+
+```plain
+00:00.0 Host bridge: Intel Corporation 440FX - 82441FX PMC [Natoma] (rev 02)
+00:01.0 ISA bridge: Intel Corporation 82371SB PIIX3 ISA [Natoma/Triton II]
+00:01.1 IDE interface: Intel Corporation 82371SB PIIX3 IDE [Natoma/Triton II]
+00:01.2 USB controller: Intel Corporation 82371SB PIIX3 USB [Natoma/Triton II] (rev 01)
+00:01.3 Bridge: Intel Corporation 82371AB/EB/MB PIIX4 ACPI (rev 03)
+00:02.0 VGA compatible controller: Cirrus Logic GD 5446
+00:03.0 Ethernet controller: Red Hat, Inc. Virtio network device
+00:04.0 SCSI storage controller: Red Hat, Inc. Virtio SCSI
+00:05.0 Communication controller: Red Hat, Inc. Virtio console
+00:06.0 3D controller: NVIDIA Corporation GA102GL [A10] (rev a1)
+00:07.0 Unclassified device [00ff]: Red Hat, Inc. Virtio memory balloon
+00:08.0 Unclassified device [00ff]: Red Hat, Inc. Virtio RNG
+```
+
+Notice the third-to-last line, which says `3D controller: NVIDIA
+Corporation`; that's good! To get much more information regarding the
```

Comment:
Since we can't always assume for certain that it is indeed the third-to-last line, I'd suggest to rephrase this as:

```suggestion
Notice the line that says `3D controller: NVIDIA
Corporation`; that's good! To get much more information regarding the
```

Reply:
Much better, yes. (And, yes, I tend to agree a lot today).